app/src/main/java/com/github/se/signify/ui/screens/profile/SettingsScreen.kt

```diff
@@ -76,165 +84,253 @@ fun SettingsScreen(
   val userName = userViewModel.userName.collectAsState()
   val profilePictureUrl = userViewModel.profilePictureUrl.collectAsState()
 
-  var newName by remember { mutableStateOf("") }
-  var selectedImageUrl by remember { mutableStateOf(profilePictureUrl.value) }
+  val focusManager = LocalFocusManager.current
+  Box(
+      modifier =
+          Modifier.fillMaxSize()
+              .clickable(
+                  onClick = { focusManager.clearFocus() },
+                  indication = null,
+                  interactionSource = remember { MutableInteractionSource() })) {
```

Comment:
I'm assuming there was no better way to have detect taps on the screen's background.